crates/forge/tests/e2e/trace_info.rs

```diff
@@ -19,19 +19,15 @@ fn trace_info_print() {
         Collected 1 test(s) from trace_info package
         Running 0 test(s) from src/
         Running 1 test(s) from tests/
-        original value: [92290246192609328148519485807009558629], converted to a string: [Entry Point Type]
-        original value: [5005878964882661740], converted to a string: [External]
-        original value: [396383589137045581302423422359764530380999520114], converted to a string: [Entry Point Selector]
-        original value: [1024936690255032842875463015856695803908824911639649964560344728797058303195]
-        original value: [4855281086078481505], converted to a string: [Calldata]
-        original value: [10]
-        original value: [433322228497498563607775107824841587], converted to a string: [Storage Address]
-        original value: [2656140848091984552801736834593776881381027675231878699139041248628012329534]
-        original value: [1366640130632659050641915078079347], converted to a string: [Caller Address]
-        original value: [469394814521890341860918960550914]
-        original value: [1242951957743815716965], converted to a string: [Call Type]
-        original value: [1130458220], converted to a string: [Call]
-        [PASS] tests::available_gas::test_trace_info_print, gas: ~1848
+
+        Entry Point Type: External
+        Selector: 0x24417f6c16e7636bef3061ce6a66f3a0bcaed4d62b43635d086869a32c204db
+        Calldata: [10]
+        Storage Address: 0x5df528bb5565afbd82e8edf5dc454f94ad15d4c1140f0661059dc33a451aa3e
+        Caller Address: 0x1724987234973219347210837402
+        Call Type: Call
```

Comment:
For debugging purposes probably would be worth to store the contract name (in case we have it) and print it too.

Reply:
Sadly it is not there anymore, I doubt it is even in sierra code of the contract